src/screens/authentication/create_new_wallet/DoubleCheckItScreen/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useEffect, useState} from 'react';
+import {ScrollView, StyleSheet} from 'react-native';
+import {ScreenProps} from "navigation/ScreenProps";
+import CreateNewWalletRouter from 'navigation/CreateNewWalletNavigation/CreateNewWalletRouter';
+import _ from 'lodash';
+import {CLayout, CHeader} from "components";
+import {scale} from "device";
+import CButton2 from "components/CButton2";
+import {useNavigation} from "@react-navigation/native";
+import {CheckItem} from "screens/authentication/create_new_wallet/components";
+import {StackNavigationProp} from "@react-navigation/stack";
+
+const numberOfRandom = 8;
+
+// @ts-ignore
+const DoubleCheckItScreen: React.FC<ScreenProps<CreateNewWalletRouter.DOUBLE_CHECK_IT_SCREEN>> = ({route}) => {
+    const {data} = route.params;
+    const [listData, setListData] = useState<any>([]);
+    const [listDataSelected, setListDataSelected] = useState<any>([]);
+
+    const {navigate} = useNavigation<StackNavigationProp<any>>();
+
+    useEffect(() => {
+        const numberOfWords = Math.floor(data.length / 3);
+
+        const randomList = _.sampleSize(data, numberOfWords);
+        let restList = getArrayNotInArray(data, randomList);
+        restList = _.shuffle(restList);
+
+        const list = randomList.map((item, index) => {
+            let listWords = [{...item, isKey: true}];
+            const randomWords = _.sampleSize(restList, 2);
+            restList = getArrayNotInArray(restList, randomWords);
+            listWords = listWords.concat(randomWords);
+            listWords = _.shuffle(listWords);
+            return listWords;
+        });
+        setListData(list);
+    }, []);
+
+    const getArrayNotInArray = (source: any, sample: any) => {
+        return source.filter((i: any) => {
+            return sample.indexOf(i) === -1;
+        })
+    };
+
+    const onSelectWords = (rowIndex: number, id: any) => {
+        if (listData && listData[rowIndex]) {
+            const listDataTemp = [...listData];
+            const listDataSelectedTemp = [...listDataSelected];
+
+            listDataTemp[rowIndex].forEach((item: any, index: number) => {
+                listDataTemp[rowIndex][index].isSelected = item.id === id;
```

Comment:
`const isSelected = item.id === id` then we can reuse the logic for if statement  bellow